### PR TITLE
Implement minireview

### LINKED ISSUE #1: GraphQL API 테스트를 위한 'minireview' 타겟 추가
### 설명

`review-web` 프로젝트의 `Cargo.toml` 파일에 `[[example]]` 타겟을 추가합니다. 이 타겟의 이름은 `minireview`이며 GraphQL API 테스트를 위해 사용됩니다. 다른 에이전트와의 연결은 REview에서만 지원되며 `minireview`에서는 동작하지 않습니다.

빌드 명령:

```sh
cargo build --example minireview
```

실행 명령:

```sh
cargo run --example minireview
```

### 할 일

- [x] `review-web` 프로젝트의 `Cargo.toml` 파일에 `minireview`라는 이름의 `[[example]]` 타겟을 추가합니다. (#13)
- [ ] `examples/minireview.rs` 파일을 만들고 REview처럼 동작할 수 있는 웹서버를 구동합니다. 구동시에는 GraphQL API 함수에서 사용할 수 있도록 `AgentManager` 트레이트를 구현하는 `struct`와 `CertManager` 트레이트를 구현하는 `struct`를 만들어서 인자로 넘겨줘야 합니다. 이 `struct`들은 mock objects이므로 실제 기능을 구현할 필요는 없고, GraphQL API 함수가 동작할 수 있는 수준만 되면 됩니다.
- [ ] `minireview`를 사용하여 GraphQL API를 테스트할 수 있는지 확인합니다.

### PR DESCRIPTION
**Pull request by sehkone**
Close #1

### CODE REVIEW

File: examples/minireview.rs [hunk 1]
```diff
@@ -1,6 +1,393 @@
...
+    fn update_certificate(
+        &self,
+        _cert: String,
+        _key: String,
+    ) -> async_graphql::Result<Vec<review_web::graphql::ParsedCertificate>, anyhow::Error> {
+        bail!("Not supported")
+    }
+}
+
+pub struct Manager {
```
Comment (msk) [inline examples/minireview.rs:59]: Does this use `db` or `store`? Shouldn't an empty `struct` suffice?

File: Cargo.toml [hunk 2]
```diff
@@ -37,8 +37,13 @@ tracing = "0.1"
 vinum = { git = "https://github.com/vinesystems/vinum.git", tag = "1.0.3" }
 
 [dev-dependencies]
+config = { version = "0.13", features = ["toml"], default-features = false }
+futures = "0.3"
+syslog = "6"
```
Comment (msk) [inline Cargo.toml:42]: Is this used?

File: examples/minireview.rs (hunk 1 above)
Comment (sehkone) [reply examples/minireview.rs]: Made it empty

File: Cargo.toml (hunk 2 above)
Comment (sehkone) [reply Cargo.toml]: Removed

**Review by msk — APPROVED** [review]

**Pull request merged**